### PR TITLE
Add leaky_relu operation

### LINKED ISSUE #1849: TIMM Leaky Relu support
### 🚀 The feature, motivation and pitch

Add `leaky_relu` definition to parser.cpp to enable `batch_norm + leaky_relu` fusion.

`PYTORCH_JIT_LOG_LEVEL=graph_fuser python benchmark.py --model cspresnet50 -b 128 --bench train --img-size 224 --amp --channels-last --torchscript --fuser nvfuser
`

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rdspring1**
Fixes #1849

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/parser.cpp [hunk 1]
```diff
@@ -2902,6 +2902,37 @@ class IrParser {
...
+
+            auto out = leaky_relu(self, negative_slope);
+            value_map.emplace(
+                node->output()->unique(), ValueHolder(out, format));
+          },
+          [](const Node* node) -> bool {
+            if (!isInputNonSizeZeroTensor(node)) {
+              return false;
+            }
+            if (node->input(1)->node()->kind() != prim::Constant) {
```
Comment (jjsjann123) [inline torch/csrc/jit/codegen/cuda/parser.cpp]: Looks like we are requiring input(1) to be constant here, but in the implementation above seems like we can take runtime input. Let's remove this line here.

**Review by jjsjann123 — CHANGES_REQUESTED** [review]
minor change requested

**Review by csarofeen — APPROVED** [review]
LGTM

**Review by jjsjann123 — APPROVED** [review]
LGTM, thanks for the update~~

**Pull request merged**